src/main/java/data/exception/DukeException.java

```diff
@@ -0,0 +1,10 @@
+package data.exception;
+
+/**
+ * Parent class of all Duke Exceptions.
+ */
+public class DukeException extends Exception{
```

Comment:
I like how the handling of exceptions are handled by various duke exception classes 👍